.github/workflows/smoke-test.yml

```diff
@@ -48,7 +44,8 @@ jobs:
 
       - name: Install
         run: |
-          sudo snap install --dangerous ./docker.snap
+          cd ./docker_*.snap
```

Comment:
Here it's a directory because as I'm using a regex for the name, the action `dawidd6/action-download-artifact` creates a repository.

Reply:
Please add this as comments in the code, so future readers also be aware of it: https://github.com/canonical/docker-snap/pull/170#discussion_r1730016927